### ISSUE TITLE
YouTube player on homepage has accessibility errors

### ISSUE THREAD

**Issue by marcysutton** [dequelabs/axe-website#8]
The YouTube player has a button with no text in it, along with a few review items. We should investigate other players so we don't have accessibility violations we can't fix.
